base/resources/scripts/languagecheck.js

```diff
@@ -0,0 +1,61 @@
+/* Nodejs to set up a server on 42536 to modify the argv.json file
+   used by in-browser VSCode. Uses the accept-language header.
+   Also used by jupyter to determine language.
+ */
+var http = require("http");
+var fs = require("fs");
+
+http
+  .createServer(function (req, res) {
```

Comment:
In the future, prefer a promise-based approach, ideally with async/await. This would involve:
```js
const fs = require("fs/promises")
http.createServer(async function(req, res) {
  // Can use await and try/catch in here.
  try {
    await somePromiseFunction();
  } catch (err) {
    // Promise was rejected...
  }
});
```
_Don't rewrite this PR; your callback approach works fine._ :smile: